src/hash/blake/mod.rs

```diff
@@ -9,6 +9,9 @@ use core::{
 #[cfg(test)]
 mod tests;
 
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
```

Comment:
One option is that instead of defining the import at the top of every module that uses serde we can reference it directly in the `cfg_attr` declaration.  See here: https://github.com/0xPolygonMiden/crypto/compare/hacka-conditional-support-for-serde...frisitano-serde-suggestion.  What do you think?

Reply:
Ah! That is better, I haven't thought about it, thanks!